pkg/logs/input/docker/parser.go

```diff
@@ -68,6 +68,8 @@ func (p *parser) Parse(msg []byte) (*message.Message, error) {
 	if idx == -1 {
 		// Nothing after the timestamp: empty message
 		return &message.Message{}, nil
```

Comment:
Maybe wrap the logic in one or two methods (either one validation function or one "empty message" function and one "empty line" function), that makes it more readable.

Your check could be higher in the parser logic (to avoid doing extra checks).

Finally, should we be a bit more protective and check for the different new lines chars ? (\n, \r and \r\n)? We could have docker running on windows (and I guess the new lines will be \r\n).